crawl-ref/source/mutation.cc

```diff
@@ -906,13 +898,13 @@ static mutation_type _get_random_mutation(mutation_type mutclass)
  */
 int mut_check_conflict(mutation_type mut, bool innate_only)
 {
-    for (const int (&confl)[3] : conflict)
+    for (const mut_conflict conflict : mut_conflicts)
```

Comment:
This should be "const mut_conflict &conflict"; otherwise you're making a copy of the mut_conflict, which you don't need to do.